### PR TITLE
Add ci-operator definitions for satellite repos

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by smarterclayton** [comment]
@sjenning @jwforres @spadgett @derekwaynecarr here's the base definitions for each of the satellite repos - creating jobs today.

File: projects/kubernetes/autoscaler.config.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      "to": "cluster-autoscaler",
+      "context_dir": "images/cluster-autoscaler/"
+    }
+  ],
+
+  "tests": [
+    {
+      "as": "unit",
+      "from": "test-bin",
+      "commands": "cd $GOPATH/src/k8s.io/autoscaler/cluster-autoscaler; go test ./..."
```
Comment (stevekuznetsov) [inline projects/kubernetes/autoscaler.config.json]: I swear we were setting the working dir in the image to point to where we cloned the code...

Comment (stevekuznetsov) [reply projects/kubernetes/autoscaler.config.json]: Or is this because of `k8s.io`

Comment (sjenning) [reply projects/kubernetes/autoscaler.config.json]: @stevekuznetsov because `k8s.io`

**Pull request merged**